### PR TITLE
Tippecanoe Improvements for atlas-checks

### PR DESCRIPTION
**Pull request by hallahan**
### Description:

There are a few small refactorings that make the line-delimited GeoJSON & tippecanoe code work better with atlas-checks.

### Potential Impact:

This PR makes the following PR possible:

https://github.com/osmlab/atlas-checks/pull/101

### Unit Test Approach:

Just refactoring and making a few things public.

### Test Results:

N/A

### CODE REVIEW

**Comment by hallahan** [comment]
cc/ @lucaspcram

File: src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java [hunk 1]
```diff
@@ -39,6 +40,55 @@ private TippecanoeCommands()
...
+     *
+     * @param geojsonDirectory
+     *            The directory to decompress.
+     */
+    public static void decompress(final Path geojsonDirectory)
+    {
+        final Time time = Time.now();
+        final String directory = geojsonDirectory.toString();
+        final String cat = String.format(
+                "find '%s' -type f -name '*.geojson.gz' -exec gzip -dr \\{\\} +", directory);
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java]: Woohoo!

File: src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java [hunk 2]
```diff
@@ -39,6 +40,55 @@ private TippecanoeCommands()
...
+     * @param geojsonDirectory
+     *            The directory of GeoJSON files to concatenate.
+     */
+    public static void concatenate(final Path geojsonDirectory)
+    {
+        final Time time = Time.now();
+        logger.info("Concatenating GeoJSON...");
+        final String directory = geojsonDirectory.toString();
+        final String cat = String.format(
+                "find '%s' -type f -name '*.geojson' \\! -path '%s'/%s -exec cat \\{\\} + > '%s/'%s",
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java]: As long as the commands calling this function are never run with super-user privileges (or even as a different user than the one invoking it), this is safe.

I am curious what you have in mind in regards to how the `TippecanoeExporter` and the `LineDelimitedGeoJSONConverter` will generally be invoked by end users? I would recommend for whatever program invokes these two commands  to at the very least sanitize its input for characters like `;`.

Comment (hallahan) [reply src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java]: Note that the method is `public static void concatenate(final Path geojsonDirectory)`. At least we know it's a `Path` getting fed in there.

Comment (lucaspcram) [reply src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java]: True. But someone could still do something like `-geojsonPath="myPath; echo vulnerable"`. However - I took a closer look and as far as I can tell, every user-provided string has been single-quoted, so I think it's actually all good!

I'm just always extra cautious about these things.

**Pull request merged**